core/lyrics/sources.go

```diff
@@ -25,17 +30,22 @@ func fromExternalFile(mf *model.MediaFile, suffix string) (model.LyricList, erro
 	contents, err := os.ReadFile(externalLyric)
 
 	if errors.Is(err, os.ErrNotExist) {
+		log.Trace(ctx, "no lyrics found at path", "path", externalLyric)
 		return nil, nil
 	} else if err != nil {
 		return nil, err
 	}
 
 	lyrics, err := model.ToLyrics("xxx", string(contents))
 	if err != nil {
+		log.Error(ctx, "error parsing lyric external file", "path", externalLyric, err)
 		return nil, err
```

Comment:
Same here. If the file is invalid, we don't want Navidrome to not try the embedded one, right?

Reply:
Error will be logged, but fallthrough instead.